modules/gitops-dex-role-mappings.adoc

```diff
@@ -0,0 +1,21 @@
+// Module is included in the following assemblies:
+//
+// * configuring-sso-for-argo-cd-on-openshift
+
+[id="dex-role-mappings_{context}"]
+= Mapping users to specific roles
+
+Argo CD cannot map to users if they have a direct `ClusterRolfeBinding` role. You can manually change users' role as `role:admin` on SSO through OpenShift.
```

Comment:
```suggestion
Argo CD cannot map to users if they have a direct `ClusterRolfeBinding` role. You can manually change users' role as `role:admin` on SSO through OpenShift.
```
Is  `ClusterRolfeBinding` correct here? Shouldn't it be  `ClusterRoleBinding` instead?